GTG/gtk/browser/taskbrowser.ui

```diff
@@ -477,86 +415,32 @@ Bertrand Rousseau (bertrand.rousseau@gmail.com)</property>
       </object>
     </child>
     <child>
-      <object class="GtkImageMenuItem" id="ctcm_mark_as_not_done">
+      <object class="GtkMenuItem" id="ctcm_mark_as_not_done">
         <property name="label" translatable="yes">Mark as Not Done</property>
         <property name="visible">True</property>
-        <property name="can_focus">False</property>
         <property name="use_underline">True</property>
-        <property name="image">ctcm_img_mark_not_done</property>
-        <property name="use_stock">False</property>
-        <property name="always_show_image">True</property>
         <signal handler="on_mark_as_done" name="activate" swapped="no"/>
       </object>
     </child>
     <child>
-      <object class="GtkImageMenuItem" id="ctcm_undismiss">
+      <object class="GtkMenuItem" id="ctcm_undismiss">
         <property name="label" translatable="yes">Und_ismiss</property>
         <property name="visible">True</property>
         <property name="can_focus">False</property>
         <property name="use_underline">True</property>
-        <property name="image">tcm_img_undismiss</property>
-        <property name="use_stock">True</property>
-        <property name="always_show_image">True</property>
         <signal handler="on_dismiss_task" name="activate" swapped="no"/>
       </object>
     </child>
     <child>
-      <object class="GtkImageMenuItem" id="ctcm_delete">
-        <property name="label">gtk-delete</property>
+      <object class="GtkMenuItem" id="ctcm_delete">
+        <property name="label">Delete</property>
```

Comment:
do you really need to remove those labels? Keeping them generic provides hidden benefits like automatic translation to the language of GNOME-environment